specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizer.json

```diff
@@ -0,0 +1,637 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Model from id": {
+            "$ref": "./examples/Analyze.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "TrainRequest": {
+      "description": "Contract to initiate a train request.",
+      "required": [
+        "source"
+      ],
+      "type": "object",
+      "properties": {
+        "source": {
+          "description": "Get or set source path.",
+          "maxLength": 2048,
+          "minLength": 0,
+          "type": "string"
+        }
+      }
+    },
+    "TrainResult": {
+      "description": "Response of the Train API call.",
+      "type": "object",
+      "properties": {
+        "modelId": {
+          "description": "Identifier of the model.",
+          "type": "string",
+          "format": "uuid",
+          "x-nullable": false
+        },
+        "trainingDocuments": {
+          "description": "List of documents used to train the model and the\r\ntrain operation error reported by each.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FormDocumentReport"
+          }
+        },
+        "errors": {
+          "description": "Errors returned during the training operation.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FormOperationError"
+          }
+        }
+      }
+    },
+    "KeysResult": {
+      "description": "Result of an operation to get\r\nthe keys extracted by a model.",
+      "type": "object",
+      "properties": {
+        "clusters": {
+          "description": "Object mapping ClusterIds to Key lists.",
+          "type": "object",
+          "additionalProperties": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": { "type": "string" }
+          }
+        }
+      }
+    },
+    "ModelResult": {
+      "description": "Result of a model status query operation.",
+      "type": "object",
+      "properties": {
+        "modelId": {
+          "description": "Get or set model identifier.",
+          "type": "string",
+          "format": "uuid",
+          "x-nullable": false
+        },
+        "status": {
+          "description": "Get or set the status of model.",
+          "enum": [ "created", "ready", "invalid" ],
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "Get or set the created date time of the model.",
+          "type": "string"
+        },
+        "lastUpdatedDateTime": {
+          "format": "date-time",
+          "description": "Get or set the model last updated datetime.",
+          "type": "string"
+        }
+      }
+    },
+    "ModelsResult": {
+      "description": "Result of query operation to fetch multiple models.",
+      "type": "object",
+      "properties": {
```

Comment:
I would recommend including a nextLink to enable pageable responses.

Reply:
Its a fair point. In our case we are limiting number of models a user can create which puts a upper bound on items returned to the caller. In our case 250 models is the limit. We wanted to  negate the need for nextLink.